pkg/cmd/github-post/main.go

```diff
@@ -44,7 +44,7 @@ const pkgEnv = "PKG"
 // 422 Validation Failed [{Resource:Issue Field:body Code:custom Message:body is too long (maximum is 65536 characters)}]
 //
 // Subtract some length just to be safe.
-const githubIssueBodyMaximumLength = 1<<16 - 1<<10
+const githubIssueBodyMaximumLength = 1<<16 - 1<<11
```

Comment:
Yeah necessary. The math also doesn't work. I'm fixing this to do the calculation for real.